src/app/components/Curation/HierarchicalGrid/index.tsx

```diff
@@ -6,29 +6,11 @@ import moment from 'moment';
 import path from 'ramda/src/path';
 import VisuallyHiddenText from '../../../legacy/psammead/psammead-visually-hidden-text/src';
 import formatDuration from '../../../lib/utilities/formatDuration';
-
 import Promo from '../../../legacy/components/Promo';
 import { DESKTOP, TABLET, MOBILE, SMALL } from './dataStructures';
 import { styles } from './index.styles';
 import { ServiceContext } from '../../../contexts/ServiceContext';
-
-type Promo = {
-  title: string;
-  description?: string;
-  type: string;
-  id: string;
-  link?: string;
-  firstPublished?: string | number;
-  duration?: string | number;
-  imageUrl?: string;
-  imageAlt?: string;
-  mediaType?: 'audio' | 'video' | 'photogallery';
-};
-
-type Promos = {
-  headingLevel: number;
-  promos: Promo[];
-};
+import { CurationGridProps } from '../types';
 
 const getStyles = (promoCount: number, i: number, mq: any) => {
```

Comment:
Could use the Theme here to type the media queries
```suggestion
const getStyles = (promoCount: number, i: number, mq: Theme['mq']) => {
```